src/stylesheets/components/navbar.scss

```diff
@@ -0,0 +1,97 @@
+@keyframes fadein {
+  0% {
+    opacity: 0%;
+  }
+  100% {
+    opacity: 100%;
+  }
+}
+
+.navbar {
+  min-height: 100vh;
```

Comment:
Given that the navbar is nested within the background, maybe setting`min-height: 100%` would be better? It makes it more flexible if we want margins or offsets for any reason